draft-ietf-taps-interface.md

```diff
@@ -1310,7 +1310,7 @@ Connection on which Clone was called, and the resulting cloned Connection. These
 connections are "entangled" with each other, and become part of a Connection
 Group. Calling Clone on any of these two Connections adds a third Connection to
 the Connection Group, and so on. Connections in a Connection Group share all
-Protocol Properties that are not applicable to a Message.
+Connection Properties except "Priority (Connection)", see {{conn-priority}}.
```

Comment:
I would prefer to keep this generic, as later documents may add Connection Properties.

Reply:
I made the text less exclusive, so now "Priority (Connection)" is an example for a non-shared Connection Property, but the text no longer implies that it is the only one.